lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -145,78 +140,30 @@ class LotForm extends React.Component {
-                                                        <option value={['SD', 'South Dakota']}>South Dakota</option>
-                                                        <option value={['TN', 'Tennessee']}>Tennessee</option>
-                                                        <option value={['TX', 'Texas']}>Texas</option>
-                                                        <option value={['UT', 'Utah']}>Utah</option>
-                                                        <option value={['VA', 'Virginia']}>Virginia</option>
-                                                        <option value={['VI', 'Virgin Islands']}>Virgin Islands</option>
-                                                        <option value={['VT', 'Vermont']}>Vermont</option>
-                                                        <option value={['WA', 'Washington']}>Washington</option>
-                                                        <option value={['WI', 'Wisconsin']}>Wisconsin</option>
-                                                        <option value={['WV', 'West Virginia']}>West Virginia</option>
-                                                        <option value={['WY', 'Wyoming']}>Wyoming</option>
-                                                    </select>
-                                                </div>
-                                                <div className="col-sm-3">
-                                                    <FormGroup label="* Zipcode" id="address_zip">
-                                                        <input type="text" className="form-control" placeholder="Zipcode" />
+                                                    <FormGroup label="State" id="address_state">
+                                                        <input type="text" className="form-control" placeholder="Kentucky" disabled value="KY" />
                                                     </FormGroup>
                                                 </div>
+                                                <div className="col-sm-3 form-group">
+                                                    <label htmlFor="address_zip" className="form-label" id="address_zip">* Zipcode</label>
+                                                    <select className="form-control" id="address_zip" onChange={formChange('address_zip')} >
+                                                        {lots.address_zip ? (
+                                                            <option value="address_zip" aria-label={`Zipcode ${lots.address_zip}`}>{lots.address_zip}</option>
+                                                        ) : (
+                                                            <option value="choose_zip" aria-label="Zipcode">Zipcode</option>
+                                                        )}
+                                                        <option value={['40515', '40515']}>40515</option>
```

Comment:
@KellyInnovation: is there a reason these zip codes are listed in this order instead of numerically sorted?